web/ascii_art_web/templates/img-upload.html

```diff
@@ -29,6 +45,8 @@ <h1>ASCII ART GENERATOR</h1>
             </div>
         </div>
     </div>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
```

Comment:
These too?

Reply:
same the bootstrap javascript linking to the CDN